test/condition.test.js

```diff
@@ -53,4 +53,10 @@ describe('Condition tests', () => {
       }
     });
   });
+  it('Null condition', async () => {
+    const cond = new Condition();
+    assert.equal('', cond.toVCL());
```

Comment:
Nothing (` `) is not a valid condition in VCL, so we have to decide what we do: resolve it to `true`, `false` or throw an error.

Reply:
I assumed that an empty condition, resolving to an empty string in VCL would be discarded by the consumer of that VCL (e.g. `vcl-utils.vcl`). Does it make more sense to return `null` instead?

Reply:
What happens when you have an empty conditions nested somewhere? I think an error is cleaner than `null` or the empty string.

Reply:
How can it be nested "somewhere"? This code applies to a top condition, only (i.e. the one declared at `condition:` in the YAML), not anywhere else.

Reply:
ok.